### DISCUSSION TITLE
Qwik + Supabase deployed on Vercel return 405 on Post Request

### DISCUSSION THREAD

**Discussion by m-caria** [vercel/community#3056] [Help]
### Summary

Hi all!

I have created a personal website with Qwik and Supabase and I have deployed the web app on Vercel.

## Problem
All work as expected but when I try to make a post request by submit a form I have a 405 error.
In my locally environment all work as expected, post request too.

## Configuration
For the Qwik configuration I've run the command `pnpm qwik add vercel-edge` and I've not change anything.
In **vercel.json** I've try to put some headers like:
```json
{
    "headers": [
	{
		"source": "/build/(.*)",
		"headers": [
			{
				"key": "Cache-Control",
				"value": "public, max-age=31536000, s-maxage=31536000, immutable"
			}
		]
	},
	{
		"source": "/api/(.*)",
		"headers": [
			{ "key": "Access-Control-Allow-Credentials", "value": "true" },
			{ "key": "Access-Control-Allow-Origin", "value": "*" },
			{
				"key": "Access-Control-Allow-Methods",
				"value": "GET,OPTIONS,PATCH,DELETE,POST,PUT"
			},
			{
				"key": "Access-Control-Allow-Headers",
				"value": "X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version"
			}
		]
	}
    ]
}
```

My form action is a simple insert to Database and a call to Supabase Edge Function:
```typescript
export const useFormAction = formAction$<NewsletterForm>(
	async (formData, event) => {
		const { error, statusText } = await subscribeToNewsletter(event, formData);
		let responseStatus: ResponseStatus = 'success';
		if (error) responseStatus = 'error';

		await sendEmail(event, {
			to: [{ email: formData.email }],
			subject: 'Subscription Confirmed',
			message: ``,
		});

		return { message: statusText, status: responseStatus };
	},
	zodForm$(newsletterFormSchema)
);
```

How can I solve this error?

Thanks in advance.

### Example

https://www.manuelcaria.dev/

### Steps to Reproduce

To reproduce the error you can fill the newsletter form or the contact form in the website and show the error in console and the post call in the network tab.

![Screenshot 2023-06-26 143551](https://github.com/vercel/community/assets/129378448/25c83449-1636-4a8b-ab4d-603d19acdee0)

**Comment by m-caria** [comment]
Any suggestion? I have been stuck for more than a week.
I have also tried asking for help in the Discord of Qwik and Supabase but to no success

**Comment by amyegan** [comment]
If you share a public [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), then someone here may be able to recognize what is going wrong
